modules/builder/package.json

```diff
@@ -14,8 +14,8 @@
     "dist"
   ],
   "dependencies": {
-    "@loaders.gl/draco": "^0.6.2",
-    "@loaders.gl/gltf": "^0.6.2",
+    "@loaders.gl/draco": "^0.7.1",
```

Comment:
Why are these on different versions? I think loaders.gl is setup so that lerna publishes synched versions of all modules, recommend sticking with that.

Reply:
@ibgreen I thought so to, but there is no @loaders.gl/draco@0.7.2
Both for **core** and **draco**
I checked with `npm info @loaders.gl/core`